lib/benchee/benchmark.ex

```diff
@@ -149,16 +165,20 @@ defmodule Benchee.Benchmark do
     end
   end
 
-  defp do_benchmark(finish_time, function, input, run_times, n, now)
-  defp do_benchmark(finish_time, _, _, run_times, _n, now)
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, now)
+  defp do_benchmark(finish_time, _, _, run_times, memory_usages, _n, now)
        when now > finish_time do
-    Enum.reverse run_times # restore correct order important for graphing
+    {Enum.reverse(run_times), memory_usages}
   end
-  defp do_benchmark(finish_time, function, input, run_times, n, _now) do
+  defp do_benchmark(finish_time, function, input, run_times, memory_usages, n, _now) do
+    :erlang.garbage_collect()
```

Comment:
My hope was that this would force a collection and give us more accurate results as a cheap MVP. However, it the tests I ran it didn't seem to make a difference, so I'll pull this out for sure.